### ISSUE TITLE
Way to install portaudio as a dependency

### ISSUE THREAD

**Issue by iamajoe** [joeferner/node-portaudio#6]
As it is for now, you have to install portaudio lib as a standalone before using this module but lets say I want to wrap an app with electron and sell it. We shouldn't ask the client to install portaudio. Dependency should already be included.

**Comment by justinmchase** [comment]
I agree, port audio code should be in a sub folder and should have a .gyp file so its easy to build this project. I'm thinking about working on doing that.
